nhaystack-rt/src/nhaystack/server/NHServerOps.java

```diff
@@ -1508,7 +1508,7 @@ public static BComponent[] getFilterComponents(NHServer server, String filter, S
       HGrid grid = hserver.readAll(filter);
       for (int i = 0; i < grid.numRows(); i++)
       {
-        HStr slotPath = (HStr) grid.row(i).get("axSlotPath", false);
+        HStr slotPath = (HStr) grid.row(i).get("n4SlotPath", false);
```

Comment:
I think the axSlotPath tag should be included here for systems that this might be applied to retrospectively.  People won't go back and change their tags on an existing system if they upgrade the module.  So for backwards compatibility it needs to include the axSlotPath tag.

Reply:
Ok now I can see it properly it looks like it's handled better.  Thanks.